inc/functions.php

```diff
@@ -36,30 +36,30 @@ function core_sitemaps_get_server() {
 	 *
 	 * @param bool $is_enabled Whether XML Sitemaps are enabled or not. Defaults to true for public sites.
 	 */
-	$is_enabled = (bool) apply_filters( 'core_sitemaps_is_enabled', $is_enabled );
+	$is_enabled = (bool) apply_filters( 'wp_sitemaps_is_enabled', $is_enabled );
 
 	if ( ! $is_enabled ) {
 		return null;
 	}
 
 	// If there isn't a global instance, set and bootstrap the sitemaps system.
-	if ( empty( $core_sitemaps ) ) {
-		$core_sitemaps = new Core_Sitemaps();
-		$core_sitemaps->init();
+	if ( empty( $wp_sitemaps ) ) {
+		$wp_sitemaps = new WP_Sitemaps();
+		$wp_sitemaps->init();
 
 		/**
-		 * Fires when initializing the Core_Sitemaps object.
+		 * Fires when initializing the Sitemaps object.
 		 *
 		 * Additional sitemaps should be registered on this hook.
 		 *
 		 * @since 5.5.0
 		 *
-		 * @param core_sitemaps $core_sitemaps Server object.
+		 * @param WP_Sitemaps $sitemaps Server object.
 		 */
-		do_action( 'core_sitemaps_init', $core_sitemaps );
+		do_action( 'wp_sitemaps_init', $sitemaps );
```

Comment:
This variable is currently undefined (see `$wp_sitemaps` above), that's why the latest test run failed.

Reply:
Forgot to rename; will fix.

Background: didn't like a `$sitemaps` global without a prefix, so I renamed it to `$wp_sitemaps`. Although no global would be even better (singleton?).

Reply:
Yeah `$wp_sitemaps` global is better. I don't have a strong opinion on global vs singleton, I feel like a singleton is like 1% better, but since core is always using globals anyway that's fine by me.